platform_setup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+sed -i -e 's/vs/vpp/g' /sonic/device/virtual/x86_64-kvm_x86_64-r0/platform_asic
```

Comment:
I had thought about it. sonic-vpp can have its own HwSKUs. It can simulate platforms with different configurations. On the other hand, there is a proposal to consolidate virtual platforms (sonic-vs, sonic-vpp and bmv2) to one and be able to switch data plane at runtime. Let's see how it evolves.

Reply:
Let's wait and see